test/db/cmd/types

```diff
@@ -132,7 +132,17 @@ e asm.lines.bb=false
 aa
 s sym.Java_com_app_ndh_NDHActivity_print
 to bins/headers/jni.h
-tl JNINativeInterface 0x464c457f
+
+# This is actually quite wrong, regarding what this function actually does :-)
```

Comment:
I suggest to add `FIXME:` or `TODO:` here, we will address this during our RzTypes work soon.

Reply:
Or just even open an issue and assign it to me.

Reply:
#1132